modules/is/templates/carbon-home/repository/conf/security/authenticators.xml.erb

```diff
@@ -49,12 +49,12 @@
             <!-- <Parameter name="ProvisioningDefaultRole">admin</Parameter> -->
             <!-- <Parameter name="IsSuperAdminRoleRequired">true</Parameter> -->
 	</Config>
-
+	
```

Comment:
Why do we need this file? Any Particular reason?

Reply:
There are few files in each repo which haven't been templated but kept in case if a user need to template it. 
IMO it is better to keep it as it is.
WDYT?

Reply:
AFAIU, user do not need to template this file. Are there any scenarios which you could think of?

Reply:
In that case, we can remove it.